src/app/providers/providers.service.ts

```diff
@@ -21,31 +22,41 @@ export class ProvidersService {
 
   constructor(
     private gitlabService: GitlabService,
-    private azureDevOpsService: AzureDevOpsService) { }
+    private gitHubService: GitHubService,
+    private azureDevOpsService: AzureDevOpsService) {
+  }
```

Comment:
nit: In TS I personally like to add comma at the end of even the last argument just so the Git diffs are more focused on what actually changed.

Also, then you can sort the parameter lines

```suggestion
    private azureDevOpsService: AzureDevOpsService,
  ) { }
```

Reply:
Ill make a separate PR to deal with this by using eslints `"comma-dangle": ["error", "always-multiline"]` as well.